src/sentry/static/sentry/app/views/settings/components/emptyMessage.tsx

```diff
@@ -33,11 +43,20 @@ const EmptyMessage = styled(
   )
 )<WrapperProps>`
   display: flex;
-  text-align: center;
-  align-items: center;
+  ${p =>
+    p.leftAligned
+      ? css`
```

Comment:
I also was using css to have a nice alignment, but after getting some feedback  I stopped using it ..are you sure you want to use them here?

Reply:
Using `css` tagged strings inside a template string doesn't do anything which is why I've been suggesting we don't do it.